### PR TITLE
add 0.2.0 release notes

### PR DESCRIPTION
**Pull request by Xiangs18**

### CODE REVIEW

File: RELEASE_NOTES.txt [hunk 1]
```diff
@@ -4,11 +4,16 @@ OVERVIEW
 -----------------------------------------
 Repo for code shared between Java services.
 
-VERSION: 0.1.1 (Release 12/21/2)
+VERSION: 0.2.0 (Release 2/7/24)
 ---------------------------------
 
-* MongoController will retry once if it encounters an IOError when deleting test files.
+* MongoController now has a constructor suited for different MongoDB versions
+* The us.kbase.common.test.controllers.shock directory was removed
```
Comment (MrCreosote) [inline RELEASE_NOTES.txt:11]:
```suggestion
* MongoController is now compatible with Mongo versions 2 through 7.
* The ShockController was removed.
```
Slight wording clarification

File: RELEASE_NOTES.txt [hunk 2]
```diff
@@ -4,11 +4,16 @@ OVERVIEW
...
+
+VERSION: 0.1.1 (Release 12/21/22)
```
Comment (MrCreosote) [inline RELEASE_NOTES.txt:13]: Thanks for fixing the date, not sure how I missed that last time

File: RELEASE_NOTES.txt (hunk 1 above)
Comment (Xiangs18) [reply RELEASE_NOTES.txt]: 👍

File: RELEASE_NOTES.txt (hunk 2 above)
Comment (Xiangs18) [reply RELEASE_NOTES.txt:13]: 👍

**Review by MrCreosote — APPROVED** [review]

**Pull request merged**